pkg/virt-api/webhooks/validating-webhook/admitters/vmi-create-admitter_test.go

```diff
@@ -1148,21 +1148,28 @@ var _ = Describe("Validating VMICreate Admitter", func() {
 		})
 
 		It("should allow BlockMultiQueue with CPU settings", func() {
-			vmi := api.NewMinimalVMI("testvm")
-			vmi.Spec.Domain.Devices.BlockMultiQueue = pointer.P(true)
-			vmi.Spec.Domain.Resources.Limits = k8sv1.ResourceList{}
-			vmi.Spec.Domain.Resources.Limits[k8sv1.ResourceCPU] = resource.MustParse("5")
+			vmi := newBaseVmi(
+				libvmi.WithLimitCPU("5"),
+				withBlockMultiQueue(),
+			)
 
-			causes := ValidateVirtualMachineInstanceSpec(k8sfield.NewPath("fake"), &vmi.Spec, config)
-			Expect(causes).To(BeEmpty())
+			ar, err := newAdmissionReviewForVMICreation(vmi)
+			Expect(err).ToNot(HaveOccurred())
+
+			resp := vmiCreateAdmitter.Admit(context.Background(), ar)
+			Expect(resp.Allowed).To(BeTrue())
+			Expect(resp.Result).To(BeNil())
 		})
 
 		It("should ignore CPU settings for explicitly rejected BlockMultiQueue", func() {
-			vmi := api.NewMinimalVMI("testvm")
```

Comment:
Noticed this test passes, even when `vmi.Spec.Domain.Devices.BlockMultiQueue = pointer.P(false)`
and yet in your helper you always make it true.
could you explain what the test is for?

Reply:
You're right to question the purpose of the test, and it was an oversight on my part. I couldn't find any validation for it in the admitter as it was removed [1], but I assume the remaining tests, while implicitly passing, aim to ensure the desired behavior in case the production code of the admitter changes.

That said, perhaps @vladikr can confirm if this understanding is correct. In any case, I'll update the helper to accept a boolean parameter.

[1] https://github.com/kubevirt/kubevirt/commit/5a2e6eabed6a958bddbd35cc2a69c5fb7aed1509